Running-Mastodon/Production-guide.md

```diff
@@ -263,19 +270,28 @@ This allows you to `sudo systemctl enable /etc/systemd/system/mastodon-*.service
 There are several tasks that should be run once a day to ensure that mastodon is
 running smoothly. As your mastodon user run `crontab -e` and enter the following
 
-```
-RAILS_ENV=production
-@daily cd /home/mastodon/live && /home/mastodon/.rbenv/shims/bundle exec rake mastodon:daily > /dev/null
-
+```sh
+    RAILS_ENV=production
+    @daily cd /home/mastodon/live && /home/mastodon/.rbenv/shims/bundle exec rake mastodon:daily > /dev/null
 ```
 
 ## Things to look out for when upgrading Mastodon
 
-You can upgrade Mastodon with a `git fetch; git checkout $(git tag | tail -n 1)` from the repository directory. You may need to run:
+If you want a stable release for production use, you should use tagged releases. To checkout the latest available tagged version:
+
+```sh
+    git clone https://github.com/tootsuite/mastodon.git
+    cd mastodon
+    git checkout $(git describe --tags `git rev-list --tags --max-count=1`)
```

Comment:
To be clear here, this PR is just moving docs over from what used to be the main repo README ... I didn't read too closely other than to preserve general organization and formatting...